lib/data.js

```
@@ -0,0 +1,177 @@
+  readLongLong() {
+    const b1 = this.readByte();
+    const b2 = this.readByte();
+    const b3 = this.readByte();
+    const b4 = this.readByte();
+    const b5 = this.readByte();
+    const b6 = this.readByte();
+    const b7 = this.readByte();
+    const b8 = this.readByte();
+    
+    if (b1 & 0x80) { // sign -> avoid overflow
+      return (((b1 ^ 0xff) * 0x100000000000000) +
```

Biome (1.9.4)

[error] This number literal will lose precision at runtime.

The value at runtime will be 72057594037927940

(lint/correctness/noPrecisionLoss)